src/lib.rs

```diff
@@ -8,6 +8,9 @@
 
 #![forbid(unsafe_code)]
 
+// dynamic trait object without `dyn` keyword is deprecated since 1.27.
+// #![allow(bare_trait_objects)]
```

Comment:
Please remove this, it's not needed any more as the MSRV is 1.27.0

Reply:
Oops, you're right.